### ISSUE TITLE
Val F1 Always -1: Right or Wrong?

### ISSUE THREAD

**Issue by xiaoruijiang** [msadat3/SciNLI#2]
Dear authors, 
Is it right to always see a -1 val F1 after more than 1000 training steps with a 32 batch_size?
train_model ...
Epoch 0 step 500 loss 0.36920568346977234 current val f1 -1
Epoch 0 step 1000 loss 0.33769387006759644 current val f1 -1

BTW, is it possible to share the finetunred SciBERT baseline model? Thanks

**Comment by xiaoruijiang** [comment]
> 

Sorry, I finally see the validation F1 improved from -1 to something. Sorry for posting the stupid question. 
But I do not know how to delete the issue I posted, ...

**Comment by msadat3** [comment]
We evaluate the model on the validation set at the end of each epoch. So, as long as the epoch number is 0, the val f1 will remain as -1. 

Sorry about the confusion.
